README.md

```diff
@@ -1,7 +1,7 @@
 
 # `container`
 
-`container` is a tool that you can use to create and run Linux containers as lightweight virtual machines on your Mac. It's written in Swift, and optimized for Apple silicon.
+`container` is a tool that you can use to create and run Linux containers as lightweight virtual machines on your Mac. It's written in Swift, and optimized for **Apple silicon**.
```

Comment:
Hi @Hemant2000-karens, thank you for the contribution! Sorry that it's not supported for Intel Macs.

I don't think a bold typeface is necessary here if we use an IMPORTANT aside below.